### DISCUSSION TITLE
Key counter is going up even during break.

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#23272] [Q&A] [answered]
I don't remember this being the case in the previous version.
Is this an intended change?

Recorded in OBS studio 29.0.2 and edited in Shotcut 22.06.23
The video **(pay attention to the key overlay)**:

https://user-images.githubusercontent.com/43719333/233673822-b1730229-4389-44a0-a1a4-6c308defe824.mp4

**Comment by DarkbitNetwork** [comment]
Reopened (now that I think about it, asking would be better than not asking.)

**Comment by Terochi** [comment]
Regressed in https://github.com/ppy/osu/pull/22654

The following code was removed from `osu.Game/Screens/Play/KeyCounterDisplay.cs` ([here](https://github.com/ppy/osu/pull/22654/files#diff-9733625e88f194b9f2c9454216823d1cb502b682f640cc57a3bf6a6c78d52eda?plain=1#L82-L93)), but in no way reimplemented
```cs
        public bool IsCounting
        {
            get => isCounting;
            set
            {
                if (value == isCounting) return;

                isCounting = value;
                foreach (var child in Children)
                    child.IsCounting = value;
            }
        }
```

**Reply by bdach** [reply]
that's just false though?

https://github.com/ppy/osu/blob/cc96d275c2f445c433507e8cf4fecf8eecab1a80/osu.Game/Screens/Play/HUD/KeyCounterDisplay.cs#L34-L37

the bindings might not be hooked up, but calling it "not reimplemented" is false

**Comment by monochrome22** [comment]
This might be out of scope, but the keycount also shouldn't increase until the first object of the map appears. That's how it works in stable.
Let me know if I should open a separate discussion or issue instead.

**Comment by DarkbitNetwork** [comment, marked as answer]
Fixed by #23309!
